frontend/src/Components/PastTravel/PastTravelForm.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import { Travel } from "../../model";
+import Button from "../Button/Button";
+
+interface PastTravelFormProps {
+  onSubmit: (travel: Array<Travel>) => void;
+}
+
+export default function PastTravelForm(props: PastTravelFormProps) {
+  const [city, setCity] = useState<string>(localStorage.getItem("city") ?? "");
+  const [startDate, setStartDate] = useState<string>(
+    localStorage.getItem("startDate") ?? ""
+  );
+  const [endDate, setEndDate] = useState<string>(
+    localStorage.getItem("endDate") ?? ""
+  );
+  const [notes, setNotes] = useState<string>(
+    localStorage.getItem("notes") ?? ""
+  );
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+  }
```

Comment:
```suggestion
```

since you only use the form to send data, you dan get rid of the second function, only need the addTravel function

Reply:
very nice Thomas, thank you very much